laborator/content/stiva/4-local-var/merge-arrays.asm

```diff
@@ -6,9 +6,9 @@
 
 section .data
 
-array_1 db 27, 46, 55, 83, 84
-array_2 db 1, 4, 21, 26, 59, 92, 105
-array_output times 12 db 0
+array_1 dd 27, 46, 55, 83, 84
+array_2 dd 1, 4, 21, 26, 59, 92, 105
+array_output times 12 dd 0
```

Comment:
You need to update the skeleton as well because it's still using byte arrays. And add some blank lines before the `array_*_lower` labels. I can't add them as suggestions because they're not part of the diff.